src/main/java/com/github/vaibhavsinha/kong/impl/helper/RetrofitServiceCreator.java

```diff
@@ -1,14 +1,16 @@
 package com.github.vaibhavsinha.kong.impl.helper;
 
 //import com.jakewharton.retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
+
 import com.github.vaibhavsinha.kong.utils.HttpsUtil;
 import okhttp3.OkHttpClient;
 import retrofit2.Retrofit;
-//import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory;
-//import retrofit2.converter.gson.GsonConverterFactory;
 
 import java.lang.reflect.Proxy;
 
+//import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)